projects/ui/src/components/Common/SelectionIndicator.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import Check from '@mui/icons-material/Check';
+import { BoxProps, Stack } from '@mui/material';
+
+import { BeanstalkPalette, FontSize } from '~/components/App/muiTheme';
+
+import { remToPx } from '~/util/UI';
+
+export type SelectionIndicatorProps = {
+  selected: boolean;
+  size?: keyof typeof FontSize | number;
+} & BoxProps;
+
+const SIZE_MULTIPLIER = 0.75;
+
+const SelectionIndicator: React.FC<SelectionIndicatorProps> = ({
+  selected,
+  size: _size = 'base',
+  sx,
+}) => {
+  const size = typeof _size === 'number' ? _size : remToPx(FontSize[_size]);
+
+  return (
+    <Stack
+      alignItems="center"
+      justifyContent="center"
+      width={size}
+      height="auto"
+      sx={{
+        aspectRatio: '1',
+        borderRadius: '100%',
+        backgroundColor: selected ? 'primary.main' : 'light.main',
+        border: '1px solid',
+        borderColor: selected ? 'primary.main' : 'text.light',
+        boxSizing: 'border-box',
+        ...sx,
+      }}
+    >
+      {selected ? (
+        <Check
+          sx={{
+            color: BeanstalkPalette.white,
+            width: size * SIZE_MULTIPLIER,
+            height: 'auto',
+          }}
+        />
+      ) : null}
+    </Stack>
```

Comment:
Looks like it's just the check icon in the top right? This strikes me as a lot of code for just the icon. Could it be reduced down to a CheckCircle with a set color?

https://mui.com/material-ui/material-icons/?query=check&selected=CheckCircle